.github/workflows/test.yml

```diff
@@ -525,64 +501,41 @@ jobs:
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
 
-      - name: Run cargo check release
+      - name: Run cargo check
         run: |
-          cargo check --workspace --all-targets --release --features rustls-tls
+          cargo groups check turborepo-libraries --features rustls-tls
 
       - name: Run cargo clippy
         run: |
-          cargo clippy --workspace --all-targets --features rustls-tls
-
-      - name: Count clippy warnings
```

Comment:
Do we no longer have any clippy warnings?

This might still be useful when clippy adds warnings in the future.

Reply:
The clippy code that checks if our warnings are decreasing was very buggy and lead to a bunch of warnings sneaking in. I'm of the view that we should avoid any warnings in the first place, and pin the Rust version. When we bump the Rust version, we should fix any new warnings too. Of course, now that the clippy checks are split between turbopack/turborepo, turbopack is welcome to do something different.